assignments/array-methods.js

```diff
@@ -55,12 +55,15 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
+fullName.foreach(function(fullName)){
+   
+};
 console.log(fullName);
 
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
+let allCaps = runners.map(     =>first_name.toUpperCase());
```

Comment:
not sure what you were doing here, but here's an example with regular function.
Example:
```
let allCaps = runners.map(function(index) { //map will take everything and put into a new array, you can put anything for the parameter.
  return index.first_name.toUpperCase(); //here, we're calling the parameter from the parenthesis and calling the first_name from each object and uppercase them before putting it into the array.
});

console.log(allCaps); //should show all first names capitalized
```